dmoj/urls.py

```diff
@@ -95,230 +94,230 @@ def exception(request):
+            path('log', organization.OrganizationRequestLog.as_view(), name='organization_requests_log'),
+            path('approved', organization.OrganizationRequestLog.as_view(states=('A',), tab='approved'),
                 name='organization_requests_approved'),
-            url(r'^rejected$', organization.OrganizationRequestLog.as_view(states=('R',), tab='rejected'),
+            path('rejected', organization.OrganizationRequestLog.as_view(states=('R',), tab='rejected'),
                 name='organization_requests_rejected'),
         ])),
 
-        url(r'^/$', lambda _, pk, slug: HttpResponsePermanentRedirect(reverse('organization_home', args=[pk, slug]))),
+        path('/', lambda _, pk, slug: HttpResponsePermanentRedirect(reverse('organization_home', args=[pk, slug]))),
     ])),
 
-    url(r'^runtimes/$', language.LanguageList.as_view(), name='runtime_list'),
-    url(r'^runtimes/matrix/$', status.version_matrix, name='version_matrix'),
-    url(r'^status/$', status.status_all, name='status_all'),
-
-    url(r'^api/', include([
-        url(r'^contest/list$', api.api_v1_contest_list),
-        url(r'^contest/info/(\w+)$', api.api_v1_contest_detail),
-        url(r'^problem/list$', api.api_v1_problem_list),
-        url(r'^problem/info/(\w+)$', api.api_v1_problem_info),
-        url(r'^user/list$', api.api_v1_user_list),
-        url(r'^user/info/([\w-]+)$', api.api_v1_user_info),
-        url(r'^user/submissions/([\w-]+)$', api.api_v1_user_submissions),
-        url(r'^user/ratings/(\d+)$', api.api_v1_user_ratings),
-        url(r'^v2/', include([
-            url(r'^contests$', api.api_v2.APIContestList.as_view()),
-            url(r'^contest/(?P<contest>\w+)$', api.api_v2.APIContestDetail.as_view()),
-            url(r'^problems$', api.api_v2.APIProblemList.as_view()),
-            url(r'^problem/(?P<problem>\w+)$', api.api_v2.APIProblemDetail.as_view()),
-            url(r'^users$', api.api_v2.APIUserList.as_view()),
-            url(r'^user/(?P<user>[\w-]+)$', api.api_v2.APIUserDetail.as_view()),
-            url(r'^submissions$', api.api_v2.APISubmissionList.as_view()),
-            url(r'^submission/(?P<submission>\d+)$', api.api_v2.APISubmissionDetail.as_view()),
-            url(r'^organizations$', api.api_v2.APIOrganizationList.as_view()),
-            url(r'^participations$', api.api_v2.APIContestParticipationList.as_view()),
-            url(r'^languages$', api.api_v2.APILanguageList.as_view()),
-            url(r'^judges$', api.api_v2.APIJudgeList.as_view()),
+    path('runtimes/', language.LanguageList.as_view(), name='runtime_list'),
+    path('runtimes/matrix/', status.version_matrix, name='version_matrix'),
+    path('status/', status.status_all, name='status_all'),
+
+    path('api/', include([
+        path('contest/list', api.api_v1_contest_list),
+        re_path(r'^contest/info/(\w+)$', api.api_v1_contest_detail),
```

Comment:
These (this and the ones below) should match the ones for the v2 endpoints. i.e. 
```suggestion
        path('contest/info/<str:contest>', api.api_v1_contest_detail),
```